cabby/model/landmark_recognition/dataset_er.py

```diff
@@ -0,0 +1,232 @@
+# coding=utf-8
+# Copyright 2020 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict
+
+from absl import logging
+import numpy as np
+import os
+import pandas as pd
+
+import torch
+
+from cabby.geo import util as gutil
+from cabby.model.text import util
+from cabby.model import util as mutil
+from cabby.geo import regions
+from cabby.model import datasets
+from transformers import DistilBertTokenizerFast
+from transformers import AutoModelForTokenClassification, AutoTokenizer
+# from keras.preprocessing.sequence import pad_sequences
+from nltk.tokenize import word_tokenize
+
+
+tokenizer = AutoTokenizer.from_pretrained("bert-base-cased")
+MAX_LEN = 256
+
+

```

Comment:
Lots of duplication with other dataset*.py  files before.  Please refactor.

Reply:
I will remove dataset.py